app/pods/components/subcases/subcases-process/template.hbs

```diff
@@ -2,17 +2,17 @@
   {{#subcases/subcase-item subcase=subcase}}
     <hr class="vlc-hr"/>
     <div class="auk-u-mb-2">
-      <p class="vlc-subline">
+      <p class="vlc-subline"  data-test-subcase-type-label>
         {{await subcase.case.type.label}}
       </p>
     </div>
     <div class="auk-u-mb-2">
-      <div class="vl-title vl-title--h5">
+      <div class="vl-title vl-title--h5"  data-test-subcase-shorttitle>
         {{subcase.shortTitle}}
       </div>
     </div>
-    <div class="auk-u-mb-4">
-      <p>
+    <div class="auk-u-mb-4" data-test-subcase-title>
+      <p class="auk-u-text-pre-line">
```

Comment:
data-test tags toevoegen zonder die te gebruiken in een test of een plaats te geven in de selectors ben ik geen fan van. Ik verwijder deze en dan kan de PR door
Mogelijk gaan we die nooit gebruiken in testen en zijn die dus nutteloos